### PR TITLE
docs(helm): Added kubeVersion to documentation.

### PR DESCRIPTION
**Pull request by eicnix**
docs(helm): Added kubeVersion to documentation and examples.

### CODE REVIEW

File: docs/charts.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ The `Chart.yaml` file is required for a chart. It contains the following fields:
 ```yaml
 name: The name of the chart (required)
 version: A SemVer 2 version (required)
+kubeVersion: A Kubernetes version the cluster needs to match (optional)
```
Comment (jascott1) [inline docs/charts.md]: Can we also describe that it is a semver here? Something like:

"Specifies the compatible version(s) of Kubernetes as a SemVer version (optional)"

**Review by jascott1 — DISMISSED** [review]

File: docs/charts.md (hunk 1 above)
Comment (bacongobbler) [reply docs/charts.md]: I’d change that to a “semver range”, as it’s specifying a range of known kubernetes versions this chart is compatible with.

Comment (eicnix) [reply docs/charts.md]: I adjusted the wording to your comments

File: docs/charts.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ The `Chart.yaml` file is required for a chart. It contains the following fields:
 ```yaml
 name: The name of the chart (required)
 version: A SemVer 2 version (required)
+kubeVersion: A SemVer range of compatible Kubernetes SemVer versions (optional)
```
Comment (jascott1) [inline docs/charts.md]: Sorry for being pedantic, could we remove the second SemVer as it seem redundant.

**Review by jascott1 — CHANGES_REQUESTED** [review]
One nit and a rebase and it should be good to go. Thanks!

File: docs/charts.md (hunk 2 above)
Comment (eicnix) [reply docs/charts.md]: No worries. I just wanted to be extra specific

**Comment by jascott1** [comment]
Thank you @eicnix Would you be able to squash this into one commit?

**Comment by eicnix** [comment]
@jascott1 I squashed the commits

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**